leshan-client-core/src/main/java/org/eclipse/leshan/client/resource/LwM2mObjectEnabler.java

```diff
@@ -55,6 +58,11 @@
  * <p>
  * In case you really need the flexibility of this interface you should consider to inherit from
  * {@link BaseObjectEnabler}.
+ * <p>
+ * If you need to destroy or stop this instance on LeshanClient stopped/destroyed,
+ * please also implement {@link Destroyable} and/or {@link Stoppable}.
```

Comment:
A little detail but I think Implementing `Stoppable` without `Startable` doesn't make too much sense. 
Because either you destroy because you don't want to be able to restart or you stop which let you able to restart.

Reply:
thanks, reworded at bd7f86d